<<<<<<< HEAD
float32 angle_min
float32 angle_max
=======
float32 angle_from_robot
float32 x
float32 y
>>>>>>> 40e59f464931446ed3aae157bdee00ca857214b4
